Core/CategoriesManager.cs

```diff
@@ -1,36 +1,33 @@
-﻿using Newtonsoft.Json;
-using System;
+﻿using System;
 using System.Collections.Generic;
-using System.IO;
 using System.Linq;
 using System.Text.RegularExpressions;
 
 namespace Core
 {
     public static class CategoriesManager
     {
-		public static Dictionary<string, Func<Transaction, bool>> Load()
+		public static IDictionary<string, Func<Transaction, bool>> BuildFilters(List<Category> categories)
```

Comment:
Codacy found an issue: [Refactor this method to use a generic collection designed for inheritance.](https://app.codacy.com/manual/siriak/MoneyManager/pullRequest?prid=5836065)